models/user.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  var user = sequelize.define('user', {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true
+    },
+    username: DataTypes.STRING,
+    password: DataTypes.STRING,
+    email: DataTypes.STRING,
+    image_id: DataTypes.INTEGER
+  }, 
+  {
+    underscored: true
+  },
+  {
+    classMethods: {
```

Comment:
The classmethods problem I experienced as well. There are better ways of accomplishing this that dont rely on Sequelize and instead use features of the language. The classMethods stuff has been removed from the latest Sequelize version, see the "class User extends Model{}..." comment here https://github.com/sequelize/sequelize/issues/5869#issuecomment-221773485

Reply:
On this note, we need to update Sequelize version and all other dependencies, for that matter, and we should do this before developing any more because the latest versions may have breaking changes, as is the case with Sequelize